CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 ## Master
 
+## 0.7.3
+
+##### Breaking
+
+* None.
+
+##### Enhancements
+
+* Updated Logo
```

Comment:
since this doesn't change any functionality, I don't think a CHANGELOG entry is necessary?